makefiles/tools/edbg-devices.inc.mk

```diff
@@ -0,0 +1,142 @@
+# Atmel SAM C/D/L/R series:
```

Comment:
this looks inefficient, all findstrings will always be checked.
Maybe link them with else?

Reply:
IMHO, the `board` should still pick this. Typically a board knows its `CPU_MODEL`, right?

Reply:
> IMHO, the board should still pick this.

No, that's what I was trying to get rid of.
The board already sets `CPU_MODEL` - that should be enough.
There is no need to further duplicate that information if it can be avoided.

> this looks inefficient, all findstrings will always be checked. Maybe link them with else?

I'll sprinkle some `else` in to speed up `make flash` :wink:

Reply:
> But I doubt that it makes much of a difference.

I couldn't measure a difference, so  probably it doesn't matter. But it *looks* inefficient, especially mot of the device types are not even supported. Maybe we just uncomment those?

Reply:
> Maybe we just uncomment those?

... and add an error case (`ifeq (, $(EDBG_DEVICE_TYPE)) $(error ...))...```